theia/templates/statefulset.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "theia.fullname" . }}
+  labels:
+    {{- include "theia.labels" . | nindent 4 }}
+spec:
+  serviceName: "{{ template "theia.fullname" . }}"
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "theia.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        dev.okteto.com/inject-token: "true"
+      labels:
+        {{- include "theia.selectorLabels" . | nindent 8 }}
+    spec:
+      volumes:
+        - name: cloudbin
+          hostPath: 
+            path: /var/okteto/cloudbin
+            type: Directory
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ include "theia.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
```

Comment:
I would remove the init container, and generate the keys on the start command of the main container

Reply:
What benefit do you see of doing it in a single command vs init container?

Reply:
I think it is the other way around. Not using an init container simplifies the manifest and there is no value on having an init container.
The init container makes sense if you want to use `ssh-keygen`, as we would remove that dependency from the main container.